helm-charts/common/guardrails-usvc/templates/configmap.yaml

```diff
@@ -14,6 +14,13 @@ data:
   SAFETY_GUARD_ENDPOINT: "http://{{ .Release.Name }}-tgi-guardrails"
   {{- end }}
   SAFETY_GUARD_MODEL_ID: {{ .Values.SAFETY_GUARD_MODEL_ID | quote }}
+  {{- if eq "LLAMA" .Values.GUARDRAIL_BACKEND }}
+  {{- else if eq "LLAMA" .Values.GUARDRAIL_BACKEND }}
+  GUARDRAILS_COMPONENT_NAME: "OPEA_LLAMA_GUARD"
+  {{- else }}
+  GUARDRAILS_COMPONENT_NAME: "OPEA_WILD_GUARD"
+  {{- cat "Invalid GUARDRAIL_BACKEND:" .Values.GUARDRAIL_BACKEND | fail }}
```

Comment:
I think only the invalid notice should be there under the else condition, and the 'GUARDRAILS_COMPONENT_NAME' definition should be move to previous condition